### ISSUE TITLE
YARD::CodeObjects::BUILTIN_EXCEPTIONS - missing exception classes

### ISSUE THREAD

**Issue by MSP-Greg** [lsegal/yard#1003]
The following root exception classes are missing from BUILTIN_EXCEPTIONS 

```ruby
ClosedQueueError
EncodingError
FiberError
KeyError
StopIteration
UncaughtThrowError
```

Replacement code (sorted)

```ruby
    BUILTIN_EXCEPTIONS = ["ArgumentError", "ClosedQueueError", "EncodingError",
      "EOFError", "Exception", "FiberError", "FloatDomainError", "IndexError",
      "Interrupt", "IOError", "KeyError", "LoadError", "LocalJumpError",
      "NameError", "NoMemoryError", "NoMethodError", "NotImplementedError",
      "RangeError", "RegexpError", "RuntimeError", "ScriptError", "SecurityError",
      "SignalException", "StandardError", "StopIteration", "SyntaxError",
      "SystemCallError", "SystemExit", "SystemStackError", "ThreadError",
      "TypeError", "UncaughtThrowError", "ZeroDivisionError"]
```

* OS: Windows
* Ruby version 2.3.2 & 2.4.0
* YARD version 0.9.5
* Relevant software dependency/versions:
  * NA

I have read the [Contributing Guide][contrib].

[contrib]: https://github.com/lsegal/yard/blob/master/CONTRIBUTING.md

**Comment by lsegal** [comment]
Can you pull request this?

**Comment by MSP-Greg** [comment]
Yes to both this and #1004.  Soon, but not immediately.

**Comment by lsegal** [comment]
Closed by #1005
